pkg/reconciler/v1alpha1/autoscaling/hpa/hpa.go

```diff
@@ -159,7 +159,7 @@ func (c *Reconciler) reconcile(ctx context.Context, key string, pa *pav1alpha1.P
 	} else if !metav1.IsControlledBy(hpa, pa) {
 		// Surface an error in the PodAutoscaler's status, and return an error.
 		pa.Status.MarkResourceNotOwned("HorizontalPodAutoscaler", desiredHpa.Name)
-		return fmt.Errorf("PodAutoscaler: %q does not own HPA: %q", pa.Name, desiredHpa.Name)
+		return nil
```

Comment:
Should we log something?

Reply:
IDK that it buys us much, since it's surfaced in the resource...